services/ows_refactored/prod_af_ows_root_cfg.py

```diff
@@ -25,6 +25,7 @@
         "allowed_urls": [
             "https://ows.digitalearth.africa",
             "https://ows-latest.digitalearth.africa",
+            "https://geoserver.digitalearth.africa/geoserver/web",
```

Comment:
Please remove the "/geoserver/web"

Reply:
Yes, thank you @SeffatChowdhury I removed the line.